### ISSUE TITLE
Mucura gen. nov. (Dennstaedtiaceae) [PASSED]

### ISSUE THREAD

**Issue by GuguYanez** [pteridogroup/ppg#4]
### Name of taxon

Mucura

### Rank of taxon

Genus

### Approximate number of species affected

two - Previously D. globulifera (Poir.) Hieron and D. bipinnata, currently Mucura bipinnata (Cav.) L.A.Triana & Sundue and Mucura globulifera (Poir.) L.A.Triana &Sundue.

### Description of change

Recognition of the new genus Mucura L.A.Triana & Sundue belonging to the Dennstaedtiaceae family that includes two species previously included in Dennstaedtia: D. globulifera (Poir.) Hieron and D. bipinnata (Cav.) Maxon.

### Reason for change

This genus of ferns was created by Triana et al. (2022) after performing a phylogeny of the subfamily Dennstaedtioideae (Dennstaedtiaceae) based on four regions of the chloroplast (rbcL coding gene, rpl16 intron, rps4 coding region along with the rps4-trnS intergenic spacer, and the trnL-F intergenic spacer). In the article the authors carried out several taxonomic changes according to the phylogenetic relationships suggested by morphological and molecular evidence and several autapomorphic states recovered through ancestral character state reconstruction. Among these, the authors transfer the species previously identified as Dennstaedtia globulifera (Poir.) Hieron and Dennstaedtia bipinnata (Cav.) Maxon to the new genus Mucura.

Mucura forms a sister monophyletic clade to the rest of the dennstaedtioid ferns. It differing from all other Dennstaedtiaceae by having rachis-costa architecture where the axes are provided with adaxial wings that are continuous between orders (from the rachis to the pinna costae, and pinna costae to pinnule costules). Mucura has a unique perispore ornamentation consisting of verrucae, broad ridges, and irregular reticles on the distal face. Also unique to Mucura are the subterete petiole bases; as far as we have seen, all other Dennstaedtioideae have petioles that are adaxially sulcate. Notably, Mucura lacks epipetiolar buds which are present in nearly all other Dennstaedtiaceae. They can further be distinguished from Dennstaedtia by their elongate, and branched rhizomes.

### Reference(s) for publication of the name

Triana‐Moreno, L. A., Yañez, A., Kuo, L. Y., Rothfels, C. J., Pena, N. T. L., Schwartsburd, P. B., & Sundue, M. (2022). Phylogenetic revision of Dennstaedtioideae (Dennstaedtiaceae: Polypodiales) with description of Mucura, gen. nov. TAXON 72(1): 20–46. https://doi.org/10.1002/tax.12858

### Code of Conduct

- [X] I agree to follow the PPG Code of Conduct

**Comment by schwartsburd** [comment]
Dennstaedtia (sensu Kramer & Green 1990, PPG I 2016) is polyphyletic. The new scheme provided by Triana-Moreno & al. (I participated as co-author) makes Dennstaedtia s.str. monophyletic, rescues Sitobolium, re-defines Microlepia, and created a new genus, Mucura, to accomodate two Neotropical species. All genera can be morphologically recognizable, and this new classification is monophyletic. It also minimized new combinations, since we proposed some type conservations. I strongly agree with this proposal, and that o Sitobolium.
